network/p2p/peerstore_provider.go

```diff
@@ -0,0 +1,43 @@
+package p2p
```

Comment:
to be reverted here

https://github.com/onflow/flow-go/pull/1814/files#diff-b55ca931ab95dac962bcdc52809f490fc06abfc9bb20aa7c1eea8fffe5dd7ccf